config/db.js

```diff
@@ -0,0 +1,30 @@
+const dotenv = require('dotenv');
```

Comment:
@murediane I think you are doing the same thing as in the environnements.js file.
Please use what we already have. You can just rename it to config.js instead of adding new files